src/GslibIO.jl

```diff
@@ -44,6 +52,25 @@ function load(file::File{format"GSLIB"})
   end
 end
 
+# helper function to parse a legacy GSLIB file
+function parse_legacy(filename::AbstractString)
+  open(filename) do fs
+    header = readline(fs)
+    # the second line contains the number of variables (the first integer) and it may contain extra info (comments)
+    linesplit = split(readline(fs))
+    nvars = parse(Int, linesplit[1])
+    # properties names
+    vars = [Symbol(strip(readline(fs))) for i in 1:nvars]
+
+    metadata = LegacySpec(header, nvars, vars)
+
+    # read data
+    data = readdlm(fs)
+
+    metadata, data
+  end
+end
```

Comment:
Very clean and easy to follow 💯 thanks for that!

Can you break the first comment into 2 short lines? Also, can you remove the comment that says "properties names"? It is clear from the next line. Also rename the variable `vars` to `varnames` to be consistent with our LegacySpec struct.

Reply:
First comment now in two lines to facilitate reading in a screen. Comment "properties names" removed. Variable `vars` renamed to `varnames`.